main/settings.py

```diff
@@ -120,7 +120,6 @@
     "mail",
     "klasses",
     "profiles",
-    "applications",
```

Comment:
I did not get it why we need to remove it, only by updating our `get_list` would fix it.